src/libheif/LibHeifRules.cmake

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (c) 2015-2019 by Gilles Caulier, <caulier dot gilles at gmail dot com>
+#
+# Redistribution and use is allowed according to the terms of the BSD license.
+# For details see the accompanying COPYING-CMAKE-SCRIPTS file.
+
+include(CheckIncludeFile)
+
+CHECK_INCLUDE_FILE(inttypes.h HAVE_INTTYPES_H)
+CHECK_INCLUDE_FILE(stddef.h HAVE_STDDEF_H)
+CHECK_INCLUDE_FILE(strings.h HAVE_STRINGS_H)
+CHECK_INCLUDE_FILE(unistd.h HAVE_UNISTD_H)
+
+if(HAVE_INTTYPES_H)
+    add_definitions(-DHAVE_INTTYPES_H)
```

Comment:
As mentioned previously I am against adding libheif directly in our repository. Another argument for that is that this cmake file is adding definitions and include directories globally to the project (instead of doing so to a specific target or source files).